WalletWasabi/Gma/QrCodeNet/Encoding/DataEncodation/ECISet.cs

```diff
@@ -150,7 +150,7 @@ internal int GetECIValueByName(string encodingName)
 			Initialize(AppendOption.NameToValue);
 		}
 
-		if (_nameToValue.TryGetValue(encodingName, out int eCIValue))
+		if (_nameToValue!.TryGetValue(encodingName, out int eCIValue))
```

Comment:
This only silences the warning.
How about
```
if (_valueToName is { } valueToName && valueToName.TryGetValue(eCIValue, out var eCIName))
```

Reply:
`_nameToValue` and `_valueToName` will always have a value, if nothing else then a default dictionary, created by the `Initialize()` method.